internal/console/kics.go

```diff
@@ -89,9 +90,17 @@ func initialize(rootCmd *cobra.Command) error {
 		"",
 		false,
 		"display only log messages to CLI output (mutually exclusive with silent)")
+	rootCmd.PersistentFlags().StringVarP(&metric,
+		"metrics",
```

Comment:
I think we should call this flag `profiling`

```suggestion
		"profiling",
```